internal/cmd/sanitize/sanitize.go

```diff
@@ -75,15 +77,86 @@ func New() *cobra.Command {
 func ignoredPatterns() []string {
 	return append(viper.GetStringSlice("exclude"),
 		// No reason to instrument these.
-		"unsafe",
-		"runtime/cgo",
 		"runtime/pprof",
 		"runtime/race",
 
-		// https://github.com/google/oss-fuzz/issues/3639
-		"syscall",
-
 		// Do not instrument our sanitizers package
 		"github.com/CodeIntelligenceTesting/gofuzz/sanitizers",
+
+		// transitive dependencies for the sanitizers module. To get these dependencies:
+		// go list -json github.com/CodeIntelligenceTesting/gofuzz/sanitizers
+		"bytes",
```

Comment:
Quite a long list of packages we're deliberately excluding from instrumentation now. Would it be advisable to try to reduce this list to a bare minimum or is this not going to affect fuzzing results?

Reply:
This list is automatically generated using `go list`. With the current state, it is unfortunately necessary to exclude all transitive dependencies of the `sanitizers` package to avoid cyclic dependencies issues. I'm experimenting with an alternative approach based on reflection that would reduce the static dependencies we have to exclude here. I'll post an update with the results

Reply:
Sounds like this warrants a section in the `README` explaining the current limitations of our approach so users won't run into side-effects they cannot fathom.

Reply:
Added this info to the docs. 

The approach with reflection would require several changes to the transformer logic since we would need to move some type checks into the call sites to avoid importing the instrumented packages from the `sanitizers` package. So, if there are no further concerns, I would suggest to merge this PR and work on this in a dedicated PR. WDYT?

Reply:
Agreed. We should get this merged and push the proposed changes in a separate PR.